server/src/index.js

```diff
@@ -0,0 +1,223 @@
+/* eslint-disable no-console */
+import 'dotenv/config';
+
+import express from 'express';
+import cors from 'cors';
+import { EventEmitter } from 'events';
+import { WebSocketServer } from 'ws';
+import { WebSocket } from 'ws';
+import { v4 as uuidv4 } from 'uuid';
+import { connectDB, syncDB } from './utils/db.js';
+import { Room } from './models/room.js';
+import { Message } from './models/message.js';
+import { error } from 'console';
```

Comment:
The import of `error` from 'console' is unnecessary since `console.error` is used directly in the code. Consider removing this import.